public/js/thirdpartyservices.js

```diff
@@ -101,6 +101,14 @@ var thirdPartyServicesJsonArray = [
     link: {
       privacy: 'https://privacy.oath.com',
       terms: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html'
+    },
+    {
+    name: 'ACRA',
+    model: 'ACRA',
+    acra: false,
+    logo: 'images/acra.png',
```

Comment:
@KerstinMaur You forgot to add the png image inside the `images` folder,

Reply:
Oh, I mixed up the Git branches. It should be here now. 😅